patches/api/0418-Add-clickable-version-on-version-command.patch

```diff
@@ -0,0 +1,37 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheMeinerLP <p.glanz@madfix.me>
+Date: Thu, 15 Jun 2023 09:48:39 +0200
+Subject: [PATCH] Add clickable version on version command
+
+
+diff --git a/src/main/java/org/bukkit/command/defaults/VersionCommand.java b/src/main/java/org/bukkit/command/defaults/VersionCommand.java
+index e40f017f87d6b6b4770501b106c76dc69ec69abb..5c8274617d1419c53cc88b6ae8430097291fc141 100644
+--- a/src/main/java/org/bukkit/command/defaults/VersionCommand.java
++++ b/src/main/java/org/bukkit/command/defaults/VersionCommand.java
+@@ -18,6 +18,10 @@ import java.util.HashSet;
+ import java.util.List;
+ import java.util.Set;
+ import java.util.concurrent.locks.ReentrantLock;
++
++import net.kyori.adventure.text.minimessage.MiniMessage;
++import net.kyori.adventure.text.minimessage.tag.Tag;
++import net.kyori.adventure.text.minimessage.tag.resolver.TagResolver;
+ import org.bukkit.Bukkit;
+ import org.bukkit.ChatColor;
+ import org.bukkit.command.CommandSender;
+@@ -89,7 +93,14 @@ public class VersionCommand extends BukkitCommand {
+ 
+     private void describeToSender(@NotNull Plugin plugin, @NotNull CommandSender sender) {
+         PluginDescriptionFile desc = plugin.getDescription();
+-        sender.sendMessage(ChatColor.GREEN + desc.getName() + ChatColor.WHITE + " version " + ChatColor.GREEN + desc.getVersion());
++        // Paper start - Helps plugins developer to get fast plugin versions in game
++        var tags = TagResolver.builder()
```

Comment:
i don#t see the point here for mini message (beyond the fact that its usage here is less than idea, why are we explicitly creating a tag resolver instead of just passing two `Placeholder.component` tag resolvers directly to deserialise.

```java
Component.text()
    .append(Component.text(name, NamedTextColor.GREEN))
    .append(Component.text(" version "))
    .append(Component.text(version, NamedTextColor.GREEN).clickEvent(ClickEvent.copyToClipboard(
        version
    )));
```
does the same job.

Reply:
I currently on it to implement it with the builder struct